thriftbp/prometheus.go

```diff
@@ -123,8 +123,8 @@ var (
 	ttlClientReplaceCounter = promauto.NewCounterVec(prometheus.CounterOpts{
 		Namespace: promNamespace,
 		Subsystem: subsystemTTLClient,
-		Name:      "connection_houskeeping_total",
-		Help:      "Total connection housekeeping (replace the connection in the background) done in thrift ttlClient",
+		Name:      "connection_housekeeping_total",
```

Comment:
Updating the name of a metric is a backwards-incompatible change. Is it worth doing this in stages to give people a chance to update dashboards, etc.?

Reply:
this is a fairly new metric so I think it should be fine and not worth it going through the steps, unless someone feels strongly.